game_eggs/minecraft/proxy/java/infrared/egg-infrared-three-server.json

```diff
@@ -0,0 +1,112 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-02-05T22:54:33+01:00",
+    "name": "Basic - Two server",
+    "author": "oulieu.lucas@gmail.com",
+    "description": "The basic installation on infrared with two server configurable via environment variables.",
+    "features": null,
+    "docker_images": {
+        "shadowner\/infrared-egg:latest": "shadowner\/infrared-egg:latest"
```

Comment:
I would prefer if you PR an image to my repo. Why can't this be downloaded in the install script and ran from the home dir?

Reply:
to the yolks repo ? 
I will do it but no tonight sadly, it's in the Todo.md 

I use my action on my repo to always fetch the latest infrared release, but I will look more into doing this step in the installation scripts, I had just no clue about how the installation script worked before finding this repo

Reply:
If it uses github for releases you can use the git script - https://github.com/parkervcp/eggs/blob/master/scripts/github_release_grabber.sh